atomicapp/external/templates/nulecule/answers.conf.sample.tpl

```diff
@@ -0,0 +1,7 @@
+[general]
+namespace = default
+provider = kubernetes
+
+[${app_name}]
+image = centos/httpd
+
```

Comment:
@cdrage.. @rtnpro and I had a discussion about this. I forget his exact argument but he was able to convince me that this is fine for now. The template is rather small and if we change the example in the future then we can easily change the template. 

@rtnpro do you remember our discussion? You may be able to reproduce some of it here.